.github/workflows/publish_images_on_manual.yml

```
@@ -0,0 +1,32 @@
+name: On Demand (Manual) Docker Image Build
```

```suggestion
name: Build and publish Docker images on demand
```

I think the file name should include `docker`. How does `publish_docker_images_on_demand.yml` sound?

---

Fixed

---

I think you missed the second half of my comment here.

> I think the file name should include `docker`. How does `publish_docker_images_on_demand.yml` sound?

---

Fixed